tests/schemas/test_schema_with_logical_types.py

```diff
@@ -26,7 +26,7 @@ def test_logical_micro_types_schema(logical_types_micro_schemas):
     """
     Test a schema with Logical Types
     """
-    a_datetime = datetime.datetime(2019, 10, 12, 17, 57, 42)
+    a_datetime = datetime.datetime.fromtimestamp(1570903062, datetime.timezone.utc)
```

Comment:
I think it is better to use something more human readable, otherwise we will have to execute the code to see what is generating and there is not too much difference with the previous approach